src/calibrator.cc

```diff
@@ -123,33 +125,89 @@ bool Calibrator::finish(int width, int height)
         return false;
     }
 
+    // ========================================================================
+    // NEW RELIABILITY FIX:
```

Comment:
Please remove the "new reliabili fix" row, and give an example of the problem that you want to solve